packages/cloudvision-connector/src/Parser.ts

```diff
@@ -216,7 +216,10 @@ function convertNotif(notif: RawNotification): ConvertedNotification {
     nanos += notif.timestamp.nanos;
   }
   nanos = nanos.padStart(9, '0');
-  convertedNotif.timestamp = parseInt(('' + notif.timestamp.seconds + nanos).slice(0, 13), 10);
+  const stringTimestamp = '' + notif.timestamp.seconds + nanos;
+  convertedNotif.timestamp = nanosEnabled
```

Comment:
you will probably need to update the test file with a nanosEnabled case to maintain 100% coverage